backend/test_lambda_function.py

```diff
@@ -3,6 +3,7 @@
 from backend import lambda_function
 
 
+### Did you want the todos in this file?
```

Comment:
If you are using these todos as comments explain what the function does, then I would use a python doc string

`
def my_function()
"""This is my doc string"""

`